MANIFEST.in

```diff
@@ -1,2 +1 @@
-include COPYING TODO lite-server.py
-include tests/*/*.py
+include COPYING lite-server.py
```

Comment:
Where is this manifest file used? Can't we just drop it? I think it at least should not include lite-server.py.